src/easylink/implementation.py

```diff
@@ -69,6 +69,11 @@ def _validate_container_exists(self, logs: list[str]) -> list[str]:
             logs.append(err_str)
         return logs
 
+    def _get_env_vars(self, implementation_config: LayeredConfigTree) -> dict[str, str]:
```

Comment:
Where does the implementation vs pipeline collision handling happen?

Reply:
implementation_config (i.e. the environment variables that come from the pipeline) just overwrites any duplicates